torch/csrc/jit/graph_executor.cpp

```diff
@@ -477,7 +477,11 @@ void GraphExecutorImplBase::run(Stack& stack) {
   logging::getLogger()->addStatValue(
       logging::runtime_counters::GRAPH_EXECUTOR_INVOCATIONS, 1.0);
 
+#ifdef USE_STATIC_DISPATCH
+  ExecutionPlan plan(graph);
```

Comment:
This is not the right place to disable optimization. recomputing the execution plan on each invocation will _recompile_ the entire interpreter bytecode on every execution.

Reply:
The better solution is to use GraphOptimizerEnabledGuard in Android wrapper, which is in https://github.com/pytorch/pytorch/pull/30285. I'm closing this PR to avoid confusion.